index.html

```diff
@@ -1 +1,5 @@
-<div></div>
+<script src="https://unpkg.com/vue/dist/vue.js"></script>
+
+<div id="app"></div>
```

Comment:
When we populate `app.js` we'll use this id to tell let our Vue instance which part of the DOM to control.